packages/suite-data/files/translations/en.json

```diff
@@ -546,7 +546,7 @@
   "TR_CONNECTED_DIFFERENT_DEVICE": "Connected a different device?",
   "TR_CONNECTED_TO_PROVIDER": "Connected to {provider} as {user}",
   "TR_CONNECTED_TO_PROVIDER_LOCALLY": "Labels saved locally",
-  "TR_CONNECT_YOUR_DEVICE": "Connect your Trezor",
+  "TR_CONNECT_YOUR_DEVICE": "Connect & unlock your Trezor",
```

Comment:
This is here to make it visible for now, it needs to be changed in translations in general.